### ISSUE TITLE
Verify handlebars minimist vuln doesn't affect servers

### ISSUE THREAD

**Issue by 6a68** [mozilla/fxa#4592]
https://npmjs.com/advisories/1179 is out today, affects handlebars and i18n-abide.

I don't think we ever pass untrusted user input to handlebars, but we should audit our code to be sure, in auth, content, and payments servers.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1386)
┆Issue Number: FXA-1386

**Comment by data-sync-user** [comment]
➤ Jody Heavener commented:

[~jhirsch@mozilla.com] So across the payments, auth, and content servers (and excluding testing and linting), it looks like our dependence on `minimist` comes from three packages: web-push, handlebars, and i18n-abide.

I did a shallow dive and found:

  * I think our usage of `handlebars` is fine. The auth server's email TXT templates do not escape any strings, but I don't see anywhere that it is rendering user input, just our own stuff (e.g. links).
  * I checked out how the auth server how is using `web-push` to send push events, and I can only see that we pass constants and programmatically generated values (e.g. device ID) to the payload.
  * It looks like `i18n-abide` is parsing the browser's Accept-Language to determine which locale to provide, but it looks like the method that parses it is handling unsanitized input fine.
